README.md

```diff
@@ -324,28 +324,30 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    Path to your model [/home/user/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]: <ENTER>
    ```
 
-5. When prompted, please choose a train profile. Train profiles are GPU specific profiles that enable accelerated training behavior. If you are on MacOS or a Linux machine without a dedicated GPU, please choose `No Profile (CPU-Only)` by hitting Enter. There are various flags you can utilize with individual `ilab` commands that will allow you to utilize your GPU if applicable.
+5. When prompted, please choose a train profile. Train profiles are GPU specific profiles that enable accelerated training behavior. If you are on MacOS or a Linux machine without a dedicated GPU, please choose `No Profile (CPU, Apple Metal, AMD ROCm)` by hitting Enter. There are various flags you can utilize with individual `ilab` commands that will allow you to utilize your GPU if applicable.
 
    ```shell
    Welcome to InstructLab CLI. This guide will help you to setup your environment.
    Please provide the following values to initiate the environment [press Enter for defaults]:
    Path to taxonomy repo [/home/user/.local/share/instructlab/taxonomy]:
    Path to your model [/home/user/.cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]:
-   Generating `/home/user/.config/instructlab/config.yaml`...
-   Please choose a train profile to use:
-   [0] No profile (CPU-only)
-   [1] A100_H100_x2.yaml
-   [2] A100_H100_x4.yaml
-   [3] A100_H100_x8.yaml
-   [4] L40_x4.yaml
-   [5] L40_x8.yaml
-   [6] L4_x8.yaml
-   Enter the number of your choice [hit enter for the default CPU-only profile] [0]:
-   Using default CPU-only train profile.
+   Generating `/home/user/.config/instructlab/config.yaml` and `/home/user/.local/share/instructlab/internal/train_configuration/profiles`...
+   Please choose a train profile to use.
+   Train profiles assist with the complexity of configuring specific GPU hardware with the InstructLab Training library.
+   Note you can still take advantage of hardware acceleration for training even if your hardware is not listed.
+   [0] No profile (CPU, Apple Metal, AMD ROCm)
+   [1] Nvidia A100/H100 x2 (A100_H100_x2.yaml)
```

Comment:
Want to know your thoughts, I'm not sure how feasible this is bc I know vRAM can be inconsistent. But what do you think about adding the vRAM options after the GPUs. 
Like 
` [1] Nvidia A100/H100 x2 (vRAM 80GB)`
`[2] Nvidia A100/H100 x4 (vRAM 40GB/80GB)` 
or something like that, that the profiles work with. 
I know that was a pain point folks were running into with the workflow. 

Im not sure if this is a larger task that is out of scope of this PR, but I can make an issue for that if you think thats better?

Reply:
Discussed with @kelbrown20 and @cdoern offline, we're going to hold off on this for now